src/python/setup.py

```diff
@@ -0,0 +1,34 @@
+#!/usr/bin/env python
+from setuptools import setup, find_packages
+
+
+def readme():
+    with open("README.md") as desc:
+        return desc.read()
+
+
+setup(
+    name="hirte",
+    version="1.0.0",
+    description="Python wrapper implementation for Hirte.",
+    long_description=readme(),
+    author="Hirte developers",
+    url="https://github.com/containers/hirte/",
+    license="GPLv2",
+    install_requires=[
+        "dasbus",
+    ],
+    package_dir={"": "."},
+    packages=find_packages("."),
+    include_package_data=True,
+    package_data={"hirte": ["py.typed"]},
+    zip_safe=True,
+    keywords="hirte python",
+    classifiers=[
+        "Programming Language :: Python",
+        "Programming Language :: Python :: 3.7",
+        "Programming Language :: Python :: 3.8",
+        "Programming Language :: Python :: 3.9",
+        "Programming Language :: Python :: 3.10",
```

Comment:
RHEL 9 contains Python 3.9 and newer, Fedora 3.7 contains Python 3.11, so shouldn't we care only about 3.9+ versions?

Reply:
Please add also Python 3.11 as it's used on the latest Fedora version